### PR TITLE
add table of OS/command for copy/symlink the runtime folder to the config

### PR DESCRIPTION
**Pull request by matan-h**
This will create a table: `os | method (copy/symlink) | command`  to simplify the `building from source` docs on the README.

### CODE REVIEW

**Comment by the-mikedavis** [comment]
I think we should not have instructions for copying. Symlinking is always preferred IMO.

These instructions also live here: https://github.com/helix-editor/helix/blob/master/book/src/install.md

**Comment by matan-h** [comment]
I add link from the README to the docs and move the table to the docs (at 3e144e039e24859a2f9c23f2a43720d36b36a8ca)

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Note: Only certain languages have indentation definitions at the moment. Check
 
 # Installation
 
-We provide packaging for various distributions, but here's a quick method to
+We provide packaging for various distributions (see [Installation docs](https://docs.helix-editor.com/install.html)), but here's a quick method to
 build from source.
```
[the remark this answers is not in the document]
Comment (archseer) [reply README.md:36]: "Packages are available for various distributions"?

File: book/src/install.md [hunk 2]
```diff
@@ -60,6 +60,11 @@ This will install the `hx` binary to `$HOME/.cargo/bin`.
 Helix also needs it's runtime files so make sure to copy/symlink the `runtime/` directory into the
 config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overriden
 via the `HELIX_RUNTIME` environment variable.
+| OS        | copy/link | command   |
+|-----------|-----------|-----------|
+|windows    |copy       |`xcopy runtime %AppData%/helix/runtime`|
+|linux/macos|symlink (preferred)|`ln -s $PWD/runtime ~/.config/helix/runtime`
+|linux/macos|copy   |`cp -r runtime ~/.config/helix/runtime`
```
Comment (archseer) [inline book/src/install.md:67]: I'd drop the copy. Also possibly drop the copy/link column altogether.

File: book/src/install.md [hunk 3]
```diff
@@ -60,6 +60,11 @@ This will install the `hx` binary to `$HOME/.cargo/bin`.
 Helix also needs it's runtime files so make sure to copy/symlink the `runtime/` directory into the
 config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overriden
 via the `HELIX_RUNTIME` environment variable.
+| OS        | copy/link | command   |
```
Comment (archseer) [inline book/src/install.md:63]: Probably needs a newline in between table and paragraph to properly parse on various markdown implementations.

File: README.md [hunk 4]
```diff
@@ -32,10 +32,10 @@ Note: Only certain languages have indentation definitions at the moment. Check
 
 # Installation
 
-We provide packaging for various distributions, but here's a quick method to
-build from source.
+Packages are available for various distributions (see [Installation docs](https://docs.helix-editor.com/install.html)).
 
-```
+If you wold like to build from source.:
```
Comment (the-mikedavis) [inline README.md:37]:
```suggestion
If you would like to build from source:

```

File: helix-term/src/health.rs [hunk 5]
```diff
@@ -230,6 +230,7 @@ pub fn language(lang_str: String) -> std::io::Result<()> {
     for ts_feat in TsFeature::all() {
         probe_treesitter_feature(&lang_str, *ts_feat)?
     }
+        println!("wiki link : https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#{}",lang_str);
```
Comment (the-mikedavis) [inline helix-term/src/health.rs:233]: It looks like this snuck in but should probably be a separate PR.

Atlassian has a good git guide on how to separate out changes with branches: https://www.atlassian.com/git/tutorials/using-branches

Comment (matan-h) [reply helix-term/src/health.rs]: By mistake, I edit `master` instead of `develop` ...

File: book/src/install.md [hunk 6]
```diff
@@ -61,6 +61,11 @@ Helix also needs it's runtime files so make sure to copy/symlink the `runtime/`
 config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overriden
 via the `HELIX_RUNTIME` environment variable.
 
+| OS        | command   |
+|-----------|-----------|
+|windows    |`xcopy runtime %AppData%/helix/runtime`|
+|linux/macos|`ln -s $PWD/runtime ~/.config/helix/runtime`
```
Comment (the-mikedavis) [inline book/src/install.md:67]: Could you copy over this table to the readme on line 51? I don't mind it being duplicated between the two

**Review by the-mikedavis — APPROVED** [review]
looks good, thanks!
